### PR TITLE
9498 Make search export accessible

### PR DESCRIPTION
**Pull request by johnatawnclementawn**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Improves keyboard accessibility of search export panel

### Issues Solved

#9498

### CODE REVIEW

File: arches/app/templates/views/components/search/search-export.htm [hunk 1]
```diff
@@ -54,25 +103,35 @@ <h4 class="">
...
-            <h2 class="">
-                <span data-bind="text: $root.translations.listItemThreeReportLink"></span>
-            </h2>
-            <h4 class="">
-                <span data-bind="text: $root.translations.listItemThreeReportLinkText"></span>
-            </h4>
+            <div class="search-export-instruction-h2 h2">
+                <span data-bind="text: $root.translations.listItemThreeReportLink" id="report-link-header"></span>
+            </div>
+            <!-- <div class="search-export-instruction-h4 h4">
```
Comment (johnatawnclementawn) [inline arches/app/templates/views/components/search/search-export.htm:132]: Moving this text to the subtitle of the switch doesn't look as nice, but it's the easiest way I thought to make it accessible.

File: arches/app/templates/views/components/search/search-export.htm [hunk 2]
```diff
@@ -131,16 +195,32 @@ <h4 class="">
...
-    <div class="search-export download" data-bind="">
+    <!-- ko if: ((total() > {{app_settings.SEARCH_EXPORT_IMMEDIATE_DOWNLOAD_THRESHOLD}}) && (celeryRunning() !== 'True') || (format() === 'html' && total() > {{app_settings.SEARCH_EXPORT_IMMEDIATE_DOWNLOAD_THRESHOLD_HTML_FORMAT}} && celeryRunning() !== 'True')) -->
+    <div 
+        id="export-warning"
+        data-bind="
+            text: ('Arches is currently running without Celery. Exports will be limited to {{app_settings.SEARCH_EXPORT_IMMEDIATE_DOWNLOAD_THRESHOLD}} records. The html option is only allowed with a selection of up to {{app_settings.SEARCH_EXPORT_IMMEDIATE_DOWNLOAD_THRESHOLD_HTML_FORMAT}} resources.'),
```
Comment (johnatawnclementawn) [inline arches/app/templates/views/components/search/search-export.htm:202]: Please see #9520

File: arches/app/templates/views/components/search/search-export.htm [hunk 3]
```diff
@@ -8,37 +8,86 @@
...
+                <span data-bind="text: $root.translations.listItemOneFormat" id="format-instruction-header"></span>
+            </div>
+            <div class="search-export-instruction-h4 h4">
+                <span data-bind="text: $root.translations.listItemOneFormatText" id="format-instruction-text"></span>
+            </div>
         </div>
         <div class="parameter">
-            <div>
-                <label data-bind="css: { 'active': format() === 'tilecsv' }, click: function () { format('tilecsv') }" class="form-radio form-normal form-text">
+            <div role="radiogroup" style="display: grid;" aria-labelledby="format-instruction-header">
```
Comment (apeters) [inline arches/app/templates/views/components/search/search-export.htm:21]: This should probably say "Export Format" as opposed to just "Format"

**Review by apeters — CHANGES_REQUESTED** [review]

**Review by ryan86 — APPROVED** [review]
LGTM

**Pull request merged**